File: Assignment51to53/program15.java
//write java program which accept number of rows and number of columns ffrom user and display below pattern.

/*
1  1  1  1 
2  2  2  2
3  3  3  3
4  4  4  4
 */
import java.lang.*;
import java.util.*;

class pattern
{
    public void pattern(int iRow,int iCol)
    {   
        int i = 0, j = 0;
        char ch = '\0',ch2 = '\0';

        for(i = 1; i<= iRow+1; i++)
        {
          for(j = 1; j<= iCol ; j++)
          {              
               System.out.print(i+"\t");
          }
          System.out.println();
        }
       
    }
}
class program15
{
    public static void main(String arg[])
    {
        pattern obj = new pattern();
        Scanner sobj = new Scanner(System.in);

      System.out.println("Enter the Row Number..");
        int iValue = sobj.nextInt();

        System.out.println("Enter the Columns Number...");
        int iValue2 = sobj.nextInt();

        obj.pattern(iValue,iValue2);
    }
}

File: Assignment3638/program15.c
//write a recursive program which accept number from user and returnits reverse number.
// Input : 523
// OutPut : 325

#include<stdio.h>

int Reverse(int iNo)
{
  static int iRev = 0;
  static int iDigit = 0;

  if(iNo != 0)
  {
    iDigit = iNo%10;
    iRev = iRev * 10 + iDigit;
    iNo = iNo/10;
    
    Reverse(iNo);
  }
  return iRev;
}

int main()
{
  int iValue = 0;
  int iRet = 0;

  printf("Enter the Number\n");
  scanf("%d",&iValue);

    iRet = Reverse(iValue);
    printf("In Reverse order : %d\n",iRet);
    return 0;
}